Cargo.toml

```diff
@@ -171,6 +171,7 @@ members = [
     "napi-dispatcher/wasm/src/proc_macros",
     "safer-ffi-build",
 ]
+exclude = ["examples/point"]
```

Comment:
```suggestion
```

This can also be achieved by adding:

```toml
[workspace]
```

at the end of `examples/point/Cargo.toml`, which I personally prefer as it's about `examples/point` "seeing _itself_ out" 😄